src/translations/translationHelper.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+
+const isRootLocaleComment =
+  reply => (reply.depth === 1 && reply.body.length === 4 && reply.body.substring(0, 2) === '# ');
+
+export const toObject = (body) => {
+  const object = {};
+  const lines = body.split(/\n/);
+  lines.forEach((line) => {
+    const message = line.split(':');
+    if (message.length === 2) {
+      object[message[0].trim()] = message[1].trim();
+    }
+  });
+  return object;
+};
+
+export const getMessages = (replies) => {
+  const localesMap = {};
+  const messages = {};
+  const sortedReplies = _.sortBy(replies, 'net_rshares');
+
+  Object.keys(sortedReplies).forEach((key) => {
```

Comment:
This code will not work. The condition "isRootLocaleComment" remove all translations replies. And the result will be { fr: {}, en: {}, de: {} } (empty objects).